example/samples/install_debian_payload_full.json

```diff
@@ -0,0 +1,63 @@
+{
+    "options": {
+        "defaults": {
+            "rootPassword": "RackHDRocks!",
+            "hostname": "rackhd",
+            "domain": "rackhd.io",
+	    "osName": "debian",
+	    "version": "stretch",
+	    "baseUrl": "dists/xenial/main/installer-amd64/current/images/netboot/ubuntu-installer/amd64",
```

Comment:
Why doese `baseUrl` include `xenial` and `ubuntu-installer`?

Reply:
ahh !